public/js/nicab.json

```diff
@@ -82,32 +82,41 @@
   },
   {
     "title": "Cookstown",
-    "address": "7 - 11 William Street\nCookstown\nCo. Tyrone\nBT80 8AX",
-    "phone": "028 8676 6891",
+    "address": "The Enterprise Centre\nDerryloran Industrial Estate, Sandholes\nCookstown\nBT80 9LU",
```

Comment:
"Derryloran Industrial Estate\nSandholes" not "Derryloran Industrial Estate, Sandhogs"